src/Dynamicweb.DataIntegration.Providers.OrderProvider.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
-		<Version>10.7.0</Version>
+		<Version>10.8.0</Version>
```

Comment:
change this to 10.8.1 as nuget already have a 10.8.0?

Reply:
Missed that, thank you!